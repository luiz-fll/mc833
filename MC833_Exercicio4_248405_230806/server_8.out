remoto: 127.0.0.1:6008
[SERVIDOR] Escutando em 127.0.0.1:6008
127.0.0.1:60654
GET / HTTP/1.0
Host: teste

127.0.0.1:60662
GET / HTTP/1.0
Host: teste

127.0.0.1:60670
GET / HTTP/1.0
Host: teste

127.0.0.1:60674
GET / HTTP/1.0
Host: teste

127.0.0.1:60690
GET / HTTP/1.0
Host: teste

127.0.0.1:60700
GET / HTTP/1.0
Host: teste

127.0.0.1:60706
GET / HTTP/1.0
Host: teste

127.0.0.1:60710
GET / HTTP/1.0
Host: teste

