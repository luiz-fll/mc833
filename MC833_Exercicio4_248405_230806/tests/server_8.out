remoto: 127.0.0.1:60008
[SERVIDOR] Escutando em 127.0.0.1:60008
127.0.0.1:36136
GET / HTTP/1.0
Host: teste

127.0.0.1:36152
GET / HTTP/1.0
Host: teste

127.0.0.1:36160
GET / HTTP/1.0
Host: teste

127.0.0.1:36172
GET / HTTP/1.0
Host: teste

127.0.0.1:36176
GET / HTTP/1.0
Host: teste

127.0.0.1:36192
GET / HTTP/1.0
Host: teste

127.0.0.1:36204
GET / HTTP/1.0
Host: teste

127.0.0.1:36206
GET / HTTP/1.0
Host: teste

127.0.0.1:36222
GET / HTTP/1.0
Host: teste

